src/content/blog/2024-10-25-rafikis-first-security-audit.mdx

```diff
@@ -14,9 +14,11 @@ tags:
 
 At the beginning of the year, we were in contact with a security and penetration testing company to do an audit of Rafiki. Even though the software is still in its early stages, it is essential to gather feedback early to build a strong foundation for the software's security. The primary goals of the assessment were to evaluate several Rafiki components: the GraphQL Admin APIs, the frontend Admin UI component, as well as our underlying [ILPv4 protocol](https://interledger.org/developers/get-started/). The assessment was done using Rafiki’s local playground, based on the [Open Source Security Testing Methodology Manual (OSSTMM)](https://www.isecom.org/research.html) and [Open Source Web Application Security Project (OWASP)](https://owasp.org/) methodologies. 
 
+The audit presented eight vulnerabilities in total, with different risk levels:
+
 ![Results of the assessment](/developers/img/blog/2024-10-25/results.png)
 
-The audit presented eight vulnerabilities in total. However, two items were not applicable to us:
+ Two items were not applicable to us:
```

Comment:
extreme nit: "two of the eight..." might be good to reinforce the context of what the reader is looking at but it's not blocking to me